docs/introduction/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Introduction to Blazor
+author: guardrex
+description: Learn how Blazor runs in the browser to execute C#/Razor code with WebAssembly and the Mono runtime in this introduction.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/introduction/index
+---
+# Introduction to Blazor
+
+By [Steve Sanderson](http://blog.stevensanderson.com), [Daniel Roth](https://github.com/danroth27), and [Luke Latham](https://github.com/guardrex)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+Blazor is an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly. Blazor provides all of the benefits of a client-side web UI framework using .NET on the client and optionally on the server.
+
+## Why use .NET for client-side apps?
+
+Web development has improved in many ways over the years, but building modern web apps still poses challenges. Using .NET in the browser offers many advantages that can help make web development easier and more productive: 
+
+* **Stability and consistency**: .NET provides standardized programming frameworks across platforms that are stable, feature-rich, and easy to use.
+* **Modern innovative languages**: .NET languages are constantly improving with innovative new language features.
+* **Industry-leading tools**: The Visual Studio product family provides a fantastic .NET development experience across platforms on Windows, Linux, and macOS.
+* **Speed and scalability**: .NET has a strong history of performance, reliability, and security for app development. Using .NET as a full-stack solution makes it easier to build fast, reliable, and secure apps.
+* **Full-stack development that leverages existing skills**: C#/Razor developers use their existing C#/Razor skills to write client-side code and share server and client-side logic among apps.
+* **Wide browser support**: Blazor runs on .NET using open web standards in the browser with no plugins and no code transpilation. It works in all modern web browsers, including mobile browsers.
+
+## Running .NET in the browser
```

Comment:
What do you think about changing this heading to: "How does it work?" or "How it works"?

Reply:
The manual calls for imperative phrasing if possible, and pronouns can be a problem because they don't convey what the subject is in a TOC listing (like over in the sidebar). How about anything along the lines of ...

> How Blazor runs C# in the browser

> How Blazor runs .NET in the browser

> How Blazor  loads and runs .NET assemblies in the browser